README.md

```diff
@@ -15,7 +15,11 @@ Unfortunately this way no longer works and command `ssh-add -K` in macOS Sierra
 ## Solution
 There is a possible solution – after usage of `ssh-add -K <key>` (it's recommended to use absolute path of keys) call command `ssh-add -A` on every startup of macOS.
 
-Just add .plist with the following content to the path `~/Library/LaunchAgents/` or create one with Lingon app (https://www.peterborgapps.com/lingon/):
+Just download this file directly:
```

Comment:
I love the curl option, but I wonder if we shouldn't still include the manual method (the path specifically)... eg.

1. curl it down
2. add a plist with the following content
3. create it with lingon app